Hmm, you're right.
```
E       AssertionError: assert {'bluetooth':...ervices': []}} == {'bluetooth':...ervices': []}}
E
E         Omitting 2 identical items, use -vv to show
E         Differing items:
E         {'storage_data': {'api_version': {'major': 99, 'minor': 99}, 'device_info': {'bluetooth_mac_address': '', 'bluetooth_proxy_feature_flags..
E
E         ...Full output truncated (2 lines hidden), use '-vv' to show
```
should we report it to the upstream?

--- a/pkgs/tools/misc/esphome/default.nix
+++ b/pkgs/tools/misc/esphome/default.nix
@@ -22,14 +22,14 @@ let
 in
 python.pkgs.buildPythonApplication rec {
   pname = "esphome";
-  version = "2025.2.1";
+  version = "2025.2.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     tag = version;
-    hash = "sha256-pi9y5o2kM1kEw8hskbVDz6/9AUi3dDlqgravRL37Tnc=";
+    hash = "sha256-UD+vpJ2hIWDNba2mEFLfhifujNCpjA33LCo4nyO+qBU=";
   };
 
   build-systems = with python.pkgs; [

